app/src/main/java/com/sunasterisk/a14day_challenge/data/model/User.kt

```diff
@@ -0,0 +1,4 @@
+package com.sunasterisk.a14day_challenge.data.model
+
+data class User(var name: String, var password: String, var bithday: String,
+                var height: Double, var weight: Double, var process: Int)
```

Comment:
những thuộc tính nào cần thay đổi giá trị về sau thì mới để var, nếu chỉ gán 1 lần trong lúc khởi tạo thì ưu tiên để val 

```suggestion
data class User(
    val name: String,
    val password: String, 
    val bithday: String,
    val height: Double, 
    val weight: Double, 
    val process: Int
)
```